packages/plugins/users-permissions/server/services/providers-registry.js

```diff
@@ -343,7 +345,7 @@ module.exports = () => {
       assert(typeof providerName === 'string', 'Provider name must be a string');
       assert(typeof provider === 'function', 'Provider callback must be a function');
 
-      providersCallbacks[providerName] = provider({ purest });
```

Comment:
why this change?

Reply:
@Boegie19 it's here by mistake, will revert it. Created another PR for this bug: https://github.com/strapi/strapi/pull/16839

Reply:
Reverted